package.json

```diff
@@ -10,8 +10,7 @@
     "url": "https://kiwix.org/"
   },
   "scripts": {
-    "mwoffliner": "node mwoffliner",
-    "mwmatrixoffliner": "node mwmatrixoffliner",
+    "mwoffliner": "./lib/cli.js",
```

Comment:
I get an error when running `npm i -g .` in the project. I'm not sure if this is specific to my environment, or a wider problem. Could you try it and check that `mwoffliner` is available globally on the command line please? 😄